### ISSUE TITLE
Test DMA transfer with small size on actual hardware

### ISSUE THREAD

**Issue by mhier** [ChimeraTK/DeviceAccess#138]
Previously, small DMA transfers (pcieuni driver) would throw a logic_error. This has been fixed as it was breaking the abstraction. A test with real hardware still needs to be done.

**Comment by killenb** [comment]
Did you try the dummy driver? It should behave like the read driver, also for DMA transfers.

**Comment by mhier** [comment]
yes, I changed the test to run through with that small dma transfer, but I didn't yet check the result. Maybe this should be fixed along. See the commit referencing this ticket (shown above).
